deepchem/data/tests/test_image_loader.py

```diff
@@ -71,12 +88,34 @@ def test_png_simple_load_with_labels(self):
         assert dataset.X.shape == (1, 768, 1024, 3)
         assert (dataset.y == np.ones((1,))).all()
 
+    def test_png_simple_load_with_label_as_image(self):
+        loader = dc.data.ImageLoader()
+        dataset = loader.create_dataset((self.face_path, self.face_path))
+        # These are the known dimensions of face.png
+        assert dataset.X.shape == (1, 768, 1024, 3)
+        assert dataset.y.shape == (1, 768, 1024, 3)
+
     def test_tif_simple_load(self):
         loader = dc.data.ImageLoader()
         dataset = loader.create_dataset(self.tif_image_path)
         # TODO(rbharath): Where are the color channels?
```

Comment:
@rbharath do we remove this comment?

Reply:
Yes, we should but fine to do in follow up PR